manager/controllers/app/fybrikapplication_controller_unit_test.go

```diff
@@ -752,6 +753,7 @@ func TestCopyData(t *testing.T) {
 	application := &app.FybrikApplication{}
 	g.Expect(readObjectFromFile("../../testdata/unittests/ingest.yaml", application)).NotTo(gomega.HaveOccurred())
 	application.Spec.Data[0].DataSetID = assetName
+	application.Spec.Data[0].Flow = taxonomy.CopyFlow
```

Comment:
why not change the flow inside the yaml?

Reply:
Was in both.  Removed the redundancy